### PR TITLE
Add R&D flag feature checks for SD card script execution

### PR DESCRIPTION
**Pull request by jlaitine**

### CODE REVIEW

**Review by t-salminen — APPROVED** [review]

**Comment by jlaitine** [comment]
Now it seems to be working, ready for review ...

**Comment by jlaitine** [comment]
Now all should be good again

File: ROMFS/px4fmu_common/init.d/rcS [hunk 1]
```diff
@@ -146,7 +153,7 @@ else
 
 		bsondump $PARAM_FILE
 
-		if [ -d "/fs/microsd" ]
+		if [ -d "/fs/microsd" -a SDCARD_SCRIPT_ENABLED = yes ]
```
Comment (pussuw) [inline ROMFS/px4fmu_common/init.d/rcS:156]: Should this be $SDCARD_SCRIPT_ENABLED ?

Comment (jlaitine) [reply ROMFS/px4fmu_common/init.d/rcS:156]: definitely, yes :)

File: src/systemcmds/rdfeat/rdfeat.cpp [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	const size_t num_flags = sizeof(rdct_feat_strings) / sizeof(rdct_feat_strings[0]);
+
+	if (argc < 2 || argc > 3) {
+		usage("Invalid number of arguments");
+		return ret;
+	}
+
+	if (!strncmp(argv[1], print_str, sizeof(print_str))) {
+		for (size_t i = 0; i < num_flags; i++) {
+			printf("%s: %s\n", rdct_feat_strings[i].s, (rdct_feat_strings[i].v & flags) != 0 ? "Enabled" : "Disabled");
```
[the remark this answers is not in the document]
Comment (jlaitine) [reply src/systemcmds/rdfeat/rdfeat.cpp:84]: Right, thanks!

**Comment by jlaitine** [comment]
Updated submodules & merging

**Pull request merged**